### DISCUSSION TITLE
Adding Custom Rolling Features to RollingFeatures list as string

### DISCUSSION THREAD

**Discussion by jrodenbergrheem** [JoaquinAmatRodrigo/skforecast#855] [Q&A] [answered]
Hi, I was wondering if their were any ways to add the custom rolling features detailed here https://skforecast.org/0.14.0/user_guides/rolling_skewness.html as a string in my forecast estimator so I could include it transformed with other statistics.... 

Here is something that may help illustrate: 

lgbm_forecaster = ForecasterRecursiveMultiSeries(regressor=LGBMRegressor(objective='tweedie',random_state=42),
                               
                               lags=[1,2,3,4,52],

                               window_features = RollingFeatures(stats=list(feats) + ['custom_feature'], # add custom feature to list alongwith built in feats 
                                                                 
                    window_sizes=list(windows)),
                    
                    encoding='ordinal_category', # allow model to use ts_id_columns as features
                    
                    forecaster_id='lgbm_forecaster')

Is this possible or is their not currently an integration yet? 

Thanks, 

Jack

**Comment by JavierEscobarOrtiz** [comment, marked as answer]
Hi @jrodenbergrheem. 

Not sure if I fully understand your comment, but we just added a new section in the user guide regarding how to add multiple classes as window features:

https://skforecast.org/0.14.0/user_guides/window-features-and-custom-features#adding-multiple-window-features

Please let me know if this is what you are looking for.

Javi

**Reply by jrodenbergrheem** [reply]
Hi @JavierEscobarOrtiz, this is precisly what I was after. Thank you!
